linux_basics_session03/steps/step01.md

```diff
@@ -5,8 +5,6 @@
 	import Execute from "components/Execute.svelte";
 </script>
 
```

Comment:
I'm removing headers here since you don't have them in the other tutorials. Otherwise it duplicates the header in the UI: 
![Screenshot 2023-09-06 at 4 31 54 PM](https://github.com/IFB-ElixirFr/sandboxbioscenarios/assets/2009870/c05f47fd-07e1-410c-b564-712830dbaf98)

Reply:
ok